lighthouse-core/test/computed/critical-request-chains-test.js

```diff
@@ -10,33 +10,55 @@
 const assert = require('assert').strict;
 const CriticalRequestChains = require('../../computed/critical-request-chains.js');
 const NetworkRequest = require('../../lib/network-request.js');
-const NetworkRecords = require('../../computed/network-records.js');
 const ComputedCrc = require('../../computed/critical-request-chains.js');
```

Comment:
Are `ComputedCrc` and `CriticalRequestChains` not the same here?

Reply:
Heh, so they are. pre-existing, but definitely will fix 👍